reapy/core/project/project.py

```diff
@@ -12,1240 +13,1303 @@
 
 class Project(ReapyObject):
 
-    """REAPER project."""
+	"""REAPER project."""

```

Comment:
I feel some of code formatters worked here.... 
I have also such problems, while modifying reapy. But, can You try to leave some way only lines that are changed?

Usually I use just an simple pep-8 formatter in very gentle mode.

Reply:
Yeah, I didn't think before converting spaces to tabs... What would be the best way to fix this mistake?

Reply:
Fixed! 0d10453